app/src/main/java/com/github/factotum_sdp/factotum/data/LoginDataSource.kt

```diff
@@ -46,7 +46,9 @@ class LoginDataSource {
             val user = User(
                 it.child("name").value as String,
                 it.child("email").value as String,
-                Role.valueOf(it.child("role").value as String)
+                Role.valueOf(it.child("role").value as String),
+                it.child("latitude").value as Double?,
```

Comment:
Does that modification need to be removed ? I see that a new node is used to send Location, and they are never written on the dispatch. Maybe I'm wrong.